CONTRIBUTING.md

```diff
@@ -161,59 +182,36 @@ The type of your commit indicates what type of change this commit is about. The
 accepted types are:
 
 - **docs**: Changes made to any static content files associated with Kong CE
-  or EE documentation (including typo fixes), the README.md or this file.
-- **feat**: A new website feature e.g. Ruby, JavaScript, HTML, or CSS changes to
-  support a new feature.
-- **fix**: A website bug fix (related to the Ruby, JavaScript, HTML, or CSS assets).
-  Typos and other fixes to the _contents_ of the documentation (markdown files) are
-  not included in this scope.
-- **style**: CSS fixes, formatting, missing semi colons, �
+  or EE documentation (including typo fixes), the README.md or this file
+- **feat**: A new website feature e.g. Ruby, JavaScript, HTML, or CSS changes
+  to support a new feature
+- **fix**: A website bug fix (related to the Ruby, JavaScript, HTML, or CSS
+  assets). Typos and other fixes to the _contents_ of the documentation
+  (markdown files) are not included in this scope
+- **style**: CSS fixes, formatting, missing semi colons, :nail_care:
 - **refactor**: A code change that neither fixes a bug nor adds a feature, and
-  is too big to be considered `chore`.
-- **chore**: Maintenance changes related to code cleaning that isn't
-  considered part of a refactor, build process updates, dependency bumps, or
-  auxiliary tools and libraries updates (npm modules, Travis-ci, etc...).
+  is too big to be considered `chore`
+- **chore**: Maintenance changes related to code cleaning that isn't considered
+  part of a refactor, build process updates, dependency bumps, or auxiliary
+  tools and libraries updates (npm modules, Travis-ci, etc...)
 
 
 ##### Scope
 
 The scope is the part of the codebase that is affected by your change. Choosing
 it is at your discretion, but here are some of the most frequent ones:
 
-- **`<ce or ee>/<section>`**: A change that affects the community edition or enterprise
-  edition docs and specifies which section.
-- **conf**: Configuration-related changes (new values, improvements...)
+- **`<ce or ee>/<section>`**: A change that affects the community edition or
+  enterprise edition docs and specifies which section.
+- **admin**: Changes related to the Admin API documentation
+- **proxy**: Changes related to the proxy documentation
+- **conf**: Changes related to the configuration file documentation (new
+  values, improvements...)
```

Comment:
admin, proxy, and conf should also have backticks?

Reply:
Yes, thanks!